draft-ietf-rtcweb-jsep.xml

```diff
@@ -407,6 +407,53 @@ setRemote(ANSWER) |                                   |
         sort of SessionDescription to produce.</t>
       </section>
 
+      <section title="Session Description control" anchor="sec.session-description-ctrl">
+	<t>JSEP defines the following objects as an API to allow the
+	application to control session descriptions: RtpTransceivers,
+	RtpSenders, and RtpReceivers.
+	</t>
+
+	<section title="RtpTransceivers" anchor="sec.rtptransceivers">
+	  <t>RtpTransceivers allow the application to control the RTP
```

Comment:
I think you should be more clear about the 1:1 relationship between transceiver and m-line.